crates/eframe/Cargo.toml

```diff
@@ -140,7 +140,7 @@ winapi = "0.3.9"
 bytemuck = "1.7"
 js-sys = "0.3"
 percent-encoding = "2.1"
-wasm-bindgen = "0.2.86"
+wasm-bindgen = "0.2.87"
```

Comment:
You should also update this in `.github/workflows/rust.yml` and `scripts/setup_web.sh`

Reply:
I'll do it in a separate PR to minimize the diff of this PR:
* https://github.com/emilk/egui/pull/3237